zbsprimarystorage/zbsprimarystorage/zbsagent.py

```
@@ -104,6 +101,61 @@ def __init__(self):
+        self.createTime = lp.createTime
+        self.userPolicy = lp.userPolicy
+        self.allocateStatus = lp.allocateStatus
+        self.capacity = lp.capacity
+        self.usedSize = lp.usedSize
+        self.allocatedSize = lp.allocatedSize
+        self.rawUsedSize = lp.rawUsedSize
+        self.rawWalUsedSize = lp.rawWalUsedSize
+        self.quota = lp.quota
+        if lp.redundanceAndPlaceMentPolicy:
+            try:
+                d = jsonobject.loads(base64.b64decode(lp.redundanceAndPlaceMentPolicy))
```

Ruff (0.8.2)

`jsonobject` may be undefined, or defined from star imports

(F405)